web_app/telegram/handlers/menu.py

```diff
@@ -0,0 +1,125 @@
+
+
+@menu_router.callback_query(F.data == "go_menu")
+async def menu(callback: types.CallbackQuery, db: Session, bot: Bot):
+    """
+    This function is called when the user clicks the "go_menu" button in Telegram.
+    It sends the user a message "Menu:" and displays a menu with buttons.
+    """
+    await callback.message.edit_text("Menu:", reply_markup=kb.menu())
+
+
+@menu_router.callback_query(F.data.startswith("notification_delete_confirm_"))
+async def delete_notification_confirm(callback: types.CallbackQuery, db: Session):
+    """
+    This function is called when the user confirms the deletion of a notification.
+    It deletes the notification from the database and sends the user a message about successful deletion.
+    """
+    # get the notification identifier
+    ident = callback.data.removeprefix("notification_delete_confirm_")
+    # delete the notification
+    stmp = delete(NotificationData).where(NotificationData.id == ident)
+    db.execute(stmp)
+    db.commit()
+    # send a message about successful deletion
+    await callback.message.edit_text("Notification deleted.", reply_markup=kb.menu())
+    return callback.answer("Deleted notification.")
+
+
+@menu_router.callback_query(F.data.startswith("notification_delete_"))
+async def delete_notification(callback: types.CallbackQuery):
+    """
+    This function is called when the user wants to delete a notification.
+    It prompts the user to confirm the deletion by displaying a confirmation button.
+    """
+    # get the notification identifier
+    ident = callback.data.removeprefix("notification_delete_")
+    await callback.message.edit_text(
+        "Are you sure you want to delete this notification? \n\n"
+        + callback.message.text,
+        reply_markup=kb.confirm_delete_subscribe(ident),
+    )
+    return callback.answer()
+
+
+@menu_router.callback_query(F.data.startswith("notification_adjust_"))
+async def delete_notification(callback: types.CallbackQuery):
+    """
+    This function is called when the user wants to adjust a notification.
+    It is not implemented yet, and it sends a message to the user that the feature is not ready.
+    """
+    ident = callback.data.removeprefix("notification_adjust_")
+    return callback.answer("Sorry, not ready yet. Stay tuned!")
+
+
+@menu_router.callback_query(F.data == "show_notifications")
+@menu_router.callback_query(F.data.startswith("notifications_"))
+async def show_notifications(callback: types.CallbackQuery, db: Session):
+    """
+    This function is called when the user wants to view their notifications.
+    It retrieves the notifications from the database and displays them to the user.
+    It also handles pagination if there are multiple notifications.
+    """
+    # get the current page
+    page = 0
+    if callback.data.startswith("notifications_"):
+        page = int(callback.data.removeprefix("notifications_"))
+    # get the current page of notifications
+    stmp = (
```

Comment:
the same here, move this code to `crud` 
Feel free to create `TelegramORM` class